packages/gestalt/src/TypeaheadInputField.css

```diff
@@ -0,0 +1,80 @@
+:root {
+  --bt: 4px;
+  --gray: #767676;
+  --lightGray: #efefef;
+  --white: #fff;
+}
+
+.input {
+  composes: accessibilityOutline from "./Focus.css";
```

Comment:
@simkessy let's share some of this CSS with `SearchField`. Looks like most of it is a copy/paste with a few differences.